pkg/block/index.go

```diff
@@ -248,6 +249,20 @@ type Stats struct {
 	// Specifically we mean here chunks with minTime == block.maxTime and maxTime > block.MaxTime. These are
 	// are segregated into separate counters. These chunks are safe to be deleted, since they are duplicated across 2 blocks.
 	Issue347OutsideChunks int
+	// OutOfOrderLabels represents the number of postings that contained out
+	// of order labels, a bug present in Prometheus 2.8.0 and below.
+	OutOfOrderLabels int
+}
+
+// PrometheusIssue5372Err returns an error if statsd indicates postings with out
```

Comment:
The wording seems a bit off here. Perhaps: `statsd indicates` -> `stats indicate` ?

Reply:
Addressed in the most recent patch